content/articles/Integrating-mongodb-with -java/index.md

```diff
@@ -0,0 +1,241 @@
+### Integrating MongoDB With Java
+
+MongoDB is a NoSQL database that stores record as documents. It uses JSON syntax and JavaScript Object Notation to store data. When integrating MongoDB with Java, it uses a different syntax.
+In this article, you'll learn how to connect MongoDB with Java using Eclipse and how to perform CRUD (create, read, update, and delete) operations using Java and the MongoDB Java Driver. MongoDB is based on a document model written in C++, JavaScript, and Python.
+
+### Table of Contents 
+- [Prerequisites](#prerequisite) 
+- [Relational Database Management Systems vs MongoDB](#Relational-Database-Management-Systems-vs-MongoDB)
+-  [Connecting MongoDB with Java](#Connecting-MongoDB-with-Java) 
+- [Creating MongoDB collection](#Creating-MongoDB-collection)
+- [Insert data into the collection](#Insert-data-into-the-collection) 
+- [Update data in the collection](#Update-data-in-the-collection)
+- [Delete data in the collection](#Delete-data-in-the-collection) 
+- [Delete the collection](#Delete-the-collection)
+- [Eclipse versus Intellij Idea in MongoDB integration with Java](#Eclipse-versus-Intellij-Idea-in-MongoDB-integration-with-java)  
+- [Conclusion](#Conclusion)
+ 
+### Prerequisite
+To follow along with this tutorial, you'll need the following: 
+- A basic understanding of the Java language
+- Eclipse IDE installed on your OS- 
+- MongoDB installed on your OS
+- MongoDB Java Connector
+- Java JDK 
+
+### Relational Database Management Systems vs MongoDB
+#### MongoDB is a non-relational database, while Relational Database Management System is a relational database.
+
+Relational databases store data in rows and columns, also known as records. A relational database works by the use of keys to connect information from multiple tables, e.g., foreign keys and primary keys. On the other hand, data get stored in a non-relational database. There are no tables, rows, primary keys, or foreign keys, as there are in a relational database. MongoDB makes use of documents, which get stored in the form of BSON documents.
+MySQL, Oracle, IBM DB2, and Microsoft SQL Server are some of the most popular RDBMS.
+MongoDB, CouchDB, CouchBase, Cassandra, HBase, Redis, Riak, and Neo4J are the popular NoSQL databases.
+
+#### MongoDB uses collections, whereas RDBMS uses tables.
+Collections are how MongoDB organizes documents. In relational databases, collections it makes use of tables.
+[Differences between collections and tables.](https://docs.mongodb.com/manual/core/databases-and-collections/)

```

Comment:
You have not addressed this suggestion.

I had asked you to explain what a collection is.

Reply:
`In relational databases, collections it makes use of tables.`

Collections makes use of tables? What do you mean? Can you brief about this?